custom/src/main/java/com/splunk/opentelemetry/SplunkConfiguration.java

```diff
@@ -71,6 +71,65 @@ public Map<String, String> defaultProperties() {
   public Config customize(Config config) {
     ConfigBuilder builder = config.toBuilder();
 
+    boolean metricsEnabled = config.getBoolean(METRICS_ENABLED_PROPERTY, false);
+    String metricsImplementation = config.getString(METRICS_IMPLEMENTATION);
+
+    if (!metricsEnabled || "micrometer".equals(metricsImplementation)) {
+      // disable upstream metrics instrumentations
+      // metrics are disabled, or we're still on the micrometer based implementation
+      addIfAbsent(builder, config, "otel.instrumentation.apache-dbcp.enabled", "false");
+      addIfAbsent(builder, config, "otel.instrumentation.c3p0.enabled", "false");
+      addIfAbsent(builder, config, "otel.instrumentation.hikaricp.enabled", "false");
+      addIfAbsent(builder, config, "otel.instrumentation.micrometer.enabled", "false");
+      addIfAbsent(builder, config, "otel.instrumentation.oracle-ucp.enabled", "false");
+      addIfAbsent(builder, config, "otel.instrumentation.oshi.enabled", "false");
+      addIfAbsent(builder, config, "otel.instrumentation.runtime-metrics.enabled", "false");
+      addIfAbsent(builder, config, "otel.instrumentation.tomcat-jdbc.enabled", "false");
+      addIfAbsent(builder, config, "otel.instrumentation.vibur-dbcp.enabled", "false");
+    }
+    if ("micrometer".equals(metricsImplementation)) {
+      // TODO: warn that micrometer metrics are deprecated
+      // log.info(
+      //     "Micrometer metrics are deprecated, OpenTelemetry metrics will become the default
+      // implementation in the next release.");
+    } else if ("opentelemetry".equals(metricsImplementation)) {
```

Comment:
Since the default value is `none`, you'll have to set the metrics exporter here to `otlp` too

Reply:
Now exporter should be set to none only when metrics are disabled or micrometer is used. Modified smoke tests to not set metrics exporter to `otlp`.